config/locales/en.yml

```diff
@@ -328,6 +328,8 @@ en:
         ask_question: "Ask a question"
         post_event: "Post an event"
         tell_story_blog: "Tell your story on the blog"
+    _node_event: 
+      rsvp: "RSVP"
       dashboard: "Dashboard"
```

Comment:
I agree with @Tlazypanda's suggestion here, it looks like _node_event is being nested inside the `dashboard` object which is not what we want.

@chicken-biryani can you please move these lines right in between these two lines `by: "by"` (315) and `_node_question` (316)? I think that should do it.

Reply:
Definitely.